cmd/kubeadm/app/master/manifests.go

```diff
@@ -54,7 +55,7 @@ const (
 )
 
 var (
-	v170 = version.MustParseSemantic("v1.7.0-alpha.0")
+	v170alpha0 = version.MustParseSemantic("v1.7.0-alpha.0")
```

Comment:
I actually prefer the v170 b/c we should only support released versions, correct?